### ISSUE TITLE
Outdated react-sortable-tree version

### ISSUE THREAD

**Issue by AstroCX1** [frontend-collective/react-sortable-tree-theme-file-explorer#12]
The dependency on react-sortable-tree is set to ^1.4.0. However, with 2.0 released, we should update the required dependency.

npm WARN react-sortable-tree-theme-file-explorer@1.1.2 requires a peer of react-sortable-tree@^1.4.0 but none is installed. You must install peer dependencies yourself.

**Comment by fritz-c** [comment]
That is true. For these themes, I think migrating away from the style loader, as was done in the main library, would be the most logical step before updating to 2.0.0, but I don’t quite have the time to make that conversion and check everything right now.
